### DISCUSSION TITLE
"-" in top level field names

### DISCUSSION THREAD

**Discussion by rnadhani-ns** [antonmedv/expr#713] [Q&A]
I have a json like:

`{
	"cs-app": "Dropbox",
	"cs-app-cci": 10,
	"cs-app-tags": ["HR", "Marketing"],
    "simple": "Dropbox"
}`

`"cs-app" == "Dropbox"` returns False as its considered cs-app as StringNode instead of IdentifierNode and `cs-app == "Dropbox"` fails to compile.

Is there anyway to support this out of box other than using Patch?

**Comment by antonmedv** [comment]
Yes.

```js
$env['cs-app'] == 'Dropbox'
```
